components/story-map/StoryMap.vue

```diff
@@ -5,7 +5,9 @@
 </template>
 
 <script>
-const apiKey = 'pk.eyJ1Ijoic2lnZ3lmIiwiYSI6ImNqZGsybjA4dTFvNnMzMnFvNXNkdXpiYXAifQ.cXeF4BQA1dKV6L--GI2Q_A'
+const apiKey = 'pk.eyJ1Ijoic2lnZ3lmIiwiYSI6ImNqZGsybjA4dTFvNnMzMnFvNXNkdXpiYXAifQ.cXeF4BQA1dKV6L--GI2Q_A';
```

Comment:
In all other components we don't use trailing commas and semicolons, maybe for consistency sake don't use them here?

Reply:
Semicolons are used in some other parts of JS code though... I'll remove them, let's later add a linting rule to enforce them everywhere.